source/slang/slang-ir-specialize.cpp

```diff
@@ -589,9 +590,6 @@ struct SpecializationContext
         case kIROp_Expand:
             return maybeSpecializeExpand(as<IRExpand>(inst));
 
-        case kIROp_ExpandTypeOrVal:
```

Comment:
Yong's note: since we are now running the `lowerExpandType` pass after IR lowering, we will no longer need to deal with `ExpandTypeOrVal` inst here. The current specialization logic to deal with expand types is not robust so we delete it.